core/src/main/scala/quasar/destination/redshift/RedshiftFlow.scala

```diff
@@ -242,7 +249,25 @@ object RedshiftFlow {
     xa <- rxa
     tableName <- Resource.liftF(ensureValidTableName[F](args.path))
     writeModeRef <- Resource.liftF(Ref.of[F, WriteMode](args.writeMode))
-  } yield new RedshiftFlow(deleteService, putService, config, tableName, xa, writeModeRef, args)
+  } yield {
+    val nameFragment = Fragment.const0(tableName)
+
+    val schemaFragment = config.schema.map(x => Fragment.const0(escape(x)))
+
+    val tblFragment = schemaFragment match {
+      case None => nameFragment ++ fr0" "
+      case Some(s) => s ++ fr0"." ++ nameFragment ++ fr0" "
```

Comment:
Can `++ fr0" "` after both `nameFragment` occurrences be dropped if `nameFragment = Fragment.const(tableName)`?

Reply:
👍